backend/awsvpc/awsvpc.go

```diff
@@ -103,6 +102,58 @@ func (m *AwsVpcBackend) Init(extIface *net.Interface, extIP net.IP) (*backend.Su
 	}
 	ec2c := ec2.New(auth, region)
 
+	if m.cfg.RouteTableID == "" {
+		log.Infof("routeTableID not passed as config parameter, attempting to detect")
+		metaPath := "network/interfaces/macs/"
+		macs, err := aws.GetMetaData(metaPath)
```

Comment:
We already have `instanceID`. Can we use `ec2.Instances([]string{instanceID}, nil)` to get `InstancesResp`. Then it should just be `resp.Reservations[0].Instances[0].SubnetId`. I haven't tried it but I wonder if it'll work and maybe cleaner than going through the mac addresses.